### DISCUSSION TITLE
Configure colour for cypress logging on Azure CI console

### DISCUSSION THREAD

**Discussion by Rubenfppinto** [cypress-io/cypress#27568] [Questions and Help]
Is it possible to configure the colours of things like the time taken for a test to run? From like red to green for example?

I have seen that we can disable all colour, but I didn't see a way to format the way we want. If it is possible, would this be with Cypress or azure itself?

![image](https://github.com/cypress-io/cypress/assets/43855814/c33b21ab-50ad-498a-8242-755b5bdda3e0)

**Comment by portyu9** [comment]
Hi @Rubenfppinto 

There isn't a Cypress configuration option for remapping individual colors in the built-in `spec` reporter, for example changing just the test-duration color from red to green.

Cypress uses Mocha-compatible reporters and the default reporter writes to stdout. Azure controls whether/how the ANSI colors are rendered, but it does not redefine the semantic color choices made by the reporter.

If you need specific coloring rules, the supported approach is to use a **custom Mocha reporter** (or another reporter that provides the formatting you want).

Otherwise the practical choices are to keep the standard reporter colors or disable color output entirely.

Please mark this answer as accepted if it helped, thank you.
